src/controllers/legacycontrollersettings.h

```diff
@@ -448,8 +588,25 @@ class LegacyControllerColorSetting
         return QJSValue(stringify());
     }
 
+#ifdef MIXXX_USE_QML
+    void setValue(const QJSValue& rawValue) override {
+        QColor value = rawValue.toVariant().value<QColor>();
```

Comment:
Can we avoid the variant round trip? What is expected? the rgb value or rgba? 
It would be nice to avoid the the term rawValue which van be anything. 

The type check of QJSValue is missing.